go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/licenseclassifier
+
+go 1.9
```

Comment:
any particular reason you're using that old of a version of go?  I can't remember the implications of the go version specified in the go.mod file, but we might want to use a more current version.

Reply:
I updated it to 1.11 because go.mod won't have any effect before then.  AFAIK, it (licenseclassifier) still works with older versions, but we probably don't need to encourage that.